build/config/config1.json

```diff
@@ -31,5 +31,9 @@
     "enabled": true,
     "address": "0.0.0.0:9000",
     "endpoint": "/metrics"
+  },
+  "logger": {
```

Comment:
if we change the name of the module to "log" that would look prettier I think.

Reply:
@okdas Please resolve comments if no further feedback/discussion is required.